internal/router/controllers/user.go

```diff
@@ -143,7 +144,7 @@ func (c *UserController) GetTableRow(rw http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	prefix := **prefixRow.Rows[0][0].(**string)
+	prefix := reflect.Indirect(reflect.ValueOf(prefixRow.Rows[0][0]))
```

Comment:
Alternative: This can also be: `prefix := (*prefixRow.Rows[0][0].(*interface{})).(string)`.
The way to think about it is that scanned results from queries are `*interface{}`, if we dereference that we should find the underlying "expected" type.